### PR TITLE
add alpine arm version

### LINKED ISSUE #5: Alpine version for x86 and ARM
Alpine version for x86 and ARM

Proposed solution by @Dashlorde 
- [ ] x86 #14 
- [ ] ARM #15

### PR DESCRIPTION
**Pull request by Dashlorde**
Fix #5

### CODE REVIEW

File: 3.4/arm_alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+COPY ./start.sh /opt/apache2/start.sh
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache wget unzip apache2 php7 postgresql-dev \
+    php7-opcache php7-pgsql php7-xmlrpc php7-gd php7-intl php7-session php7-json \
+    php7-iconv php7-xml php7-apache2 php7-curl php7-soap php7-openssl php7-ctype php7-zip \
+    php7-zlib php7-simplexml php7-dom php7-xmlreader php7-fileinfo php7-mbstring php7-tokenizer
+
+RUN wget -O /var/www/moodle-${version}.tar.gz https://github.com/moodle/moodle/archive/v${version}.tar.gz && \
```
Comment (empeje) [inline 3.4/arm_alpine/Dockerfile]: could you delete the artifact after extraction?

File: 3.4/arm_alpine/config/php.ini [hunk 2]
```diff
@@ -0,0 +1,1936 @@
+[PHP]
```
Comment (empeje) [inline 3.4/arm_alpine/config/php.ini]: Can you minify this by delete comment?

File: 3.4/arm_alpine/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+moodledb_alpine:
+  image: arm32v7/postgres
+  container_name: moodledb_alpine_arm
+  ports:
+    - "4000:5432"
+  environment:
+    - POSTGRES_DATABASE=moodle
```
Comment (empeje) [inline 3.4/arm_alpine/docker-compose.yml]: Put comment here like https://github.com/ole-vi/moodole/pull/16

File: 3.4/arm_alpine/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    - POSTGRES_PASSWORD=moodle
+moodle_alpine:
+  build: .
+  container_name: moodle_alpine_arm
+  ports:
+    - "8090:80"
+  links:
+    - moodledb_alpine
+  environment:
+    - MOODLE_URL=http://docker.ole.org:80
```
Comment (empeje) [inline 3.4/arm_alpine/docker-compose.yml]: We have new convention see https://github.com/ole-vi/moodole/pull/16 and add also config like this https://github.com/ole-vi/moodole/blob/e65dd944d14e26a802364bfab8131a77af8e2623/3.4/x86/config.php

**Review by empeje — CHANGES_REQUESTED** [review]
Need some changes, due to use case requirement

* Need to access site from different way
  - IP
  - Domain
  - local
  - Tunnel

follow convention written here https://github.com/ole-vi/moodole/pull/16

@Dashlorde how many MB the resulted image would be?

**Comment by empeje** [comment]
Please adapt with this changes https://github.com/ole-vi/moodole/pull/26

**Comment by empeje** [comment]
Congrats @Dashlorde ! Will test it soon.

**Review by empeje — CHANGES_REQUESTED** [review]
found it  @Dashlorde 

```
bash: line 80: deploy_arm_alpine: command not found
```
https://api.travis-ci.org/v3/job/332958913/log.txt

seems typo here
https://github.com/treehouses/moodole/blob/6bea83655a2765e0ccfbd93233f571095e9a43ea/.travis/travis_utils.sh#L126

**Comment by Dashlorde** [comment]
typo fixed @mappuji

**Comment by empeje** [comment]
Thanks @Dashlorde , last step is to test it.

**Comment by Dashlorde** [comment]
hi @mappuji I am running moodle at docker.ole.org

**Comment by empeje** [comment]
hello @Dashlorde currently checking it

**Comment by empeje** [comment]
Can you also update README to include how to use alpine version in another PR?

**Comment by empeje** [comment]
Done, will merge it.

**Pull request merged**